scripts/core/ui/utils.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import {Store} from 'redux';
+import {Provider} from 'react-redux';
+import {render, unmountComponentAtNode} from 'react-dom';
+
+interface IStoreScope extends ng.IScope {
+    store: Store<{}>;
+}
+
+/**
+ * Render react component and connect it to redux store
+ */
+export function renderComponent(component: React.ReactElement<any>, scope: IStoreScope, elem: Array<Element>) {
```

Comment:
Doesn't `reactToAngular1` do the same thing? You could wrap the initial component in the `<Provider>` and pass store as a prop.

Reply:
not sure how to do it, it works with `reactToAngular1(Provider, ['store'])` but not sure how to pass that children component

Reply:
Instead of doing `reactToAngular1(Provider, ['store'])`, do `reactToAngular1(MyCustomComponent, ['customProps'])` and use `Provider` inside `MyCustomComponent`. You would do so for each individual component. `reactToAngular1` is more like `AttachmentsListDirective` which isn't doing much more than making it possible to use a React component as a directive.

Reply:
you can give it a shot, I can't make it work

Reply:
https://github.com/tomaskikutis/superdesk-client-core/tree/pr-2832

Tests pass and UI seems to work, so I think you can merge it from my branch.